File: InsuranceCompany.Client/src/app/models/InsuredPerson.ts
import { Client } from 'src/app/models/Client';

export class InsuredPerson {
    id : string;
    client : Client;
    isMainInsuredPerson : boolean;
    isAccoridionOpen: boolean; // дополнительная переменная разворачивающегося списка

    constructor(id: string) {
        this.id = id;
    }
}

File: InsuranceCompany.MobileClient/InsuranceCompany.MobileClient/Models/InsuredPerson.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InsuranceCompany.MobileClient.Models
{
    public class InsuredPerson
    {
        public Guid? Id { get; set; }
        public bool? IsMainInsuredPerson { get; set; }

        public Guid? ClientId { get; set; }
        public Guid? InsuranceRequestId { get; set; }

        public virtual Client Client { get; set; }
    }
}
